### PR TITLE
feat:add description of wind gusts

### PR DESCRIPTION
**Pull request by sandipa12**
Displays the description of wind gust if the wind speed is greater than 15 mph.
Demo of how it displays the wind gust information:
Screen Shot 2021-06-28 at 11 17 45 AM

### CODE REVIEW

**Review by bbazyk — COMMENTED** [review]
Are there plans for more commits where this PR will display gusts to the widget? Or is this ticket specifically for retrieval from openWeather ?
P.S. I'm excited for all the changes we've made to the weather widget so far - I think the visuals and information organization has been appreciated by anyone passing the helios projector screen in the pvd office.

File: app/javascript/components/widgets/weather/current-weather.jsx [hunk 1]
```diff
@@ -32,6 +34,7 @@ const CurrentWeather = ({ weather, useLargeIcon }) => {
         <SmallSkyIcon icon={icon} />
       )}
       {description}
+      {Gust}
```
Comment (chewnoill) [inline app/javascript/components/widgets/weather/current-weather.jsx:37]: Better to stick to regular JSX syntax here I think:

```jsx
Gust {weather.current.windGusts}
```

File: app/javascript/components/widgets/weather/current-weather.jsx [hunk 2]
```diff
@@ -22,8 +23,16 @@ const IconWrapper = styled.div`
   justify-content: center;
 `;
 
+function isGusty(weather) {
```
Comment (chewnoill) [inline app/javascript/components/widgets/weather/current-weather.jsx:26]: convention is that a function that starts with `is` returns a boolean.

File: app/javascript/components/widgets/weather/current-weather.jsx (hunk 1 above)
Comment (chewnoill) [reply app/javascript/components/widgets/weather/current-weather.jsx:44]: Maybe we can inline this?
```jsx
{isGusty(weather)}
```

Comment (chewnoill) [reply app/javascript/components/widgets/weather/current-weather.jsx:44]: or even better 
```jsx
{weather.current.windGust > 15 && ` , gusty wind at ${weather.current.windGust} mph`}
```

File: app/lib/clients/weather_client.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 class Clients::WeatherClient
-  WEATHER_API_URL = 'https://api.openweathermap.org/data/2.5/onecall?lat=%{lat}&lon=%{lon}&appid=%{key}'.freeze
+  WEATHER_API_URL = 'https://api.openweathermap.org/data/2.5/onecall?lat=%{lat}&lon=%{lon}&appid=%{key}&units=imperial'.freeze
```
Comment (chewnoill) [inline app/lib/clients/weather_client.rb:2]: unclear why we're touching this. Maybe put it in its own commit? I think this is a separate idea.

Comment (sandipa12) [reply app/lib/clients/weather_client.rb:2]: The original WEATHER_API_URL has standard unit which gives the data in m/s. Adding units= imperial gives the data in mph.

File: app/javascript/components/widgets/weather/current-weather.jsx (hunk 1 above)
Comment (cbarber) [reply app/javascript/components/widgets/weather/current-weather.jsx:44]: Nit: prefer extracting/naming components over inlining code in JSX.

File: app/lib/clients/weather_client.rb (hunk 3 above)
Comment (cbarber) [reply app/lib/clients/weather_client.rb:2]: This should be a separate fix commit.
